### PR TITLE
Fixes PR template

### LINKED ISSUE #71685: PR template is slightly unclear
**What happened**: The PR template is a little unclear to new contributors.

EG:
```md
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
```

If I understand what "uncomment" is, and leave it on it's own line I get:
```
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
 /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
```
which has two problems:
- I probably left the space after the `>` (so the bot regex won't match)
- I probably didn't put a newline before the uncommented line, which is necessary for it to not just continue to be a comment block:

---

**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
 /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

--- 

vs

---

**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

---

Another issue is in this bit:
```
**Does this PR introduce a user-facing change?**:
<!--  
If no, just write "NONE".
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
2. 
-->
```release-note

```

We should probably make this something like a literal:
```release-note
NONE
```

If you're not an experienced contributor, it's not very clear where you should write "NONE" and what kind of quoting you should use, etc.

**What you expected to happen**: I hope for the PR template to be a little more obvious and easy to use

**How to reproduce it (as minimally and precisely as possible)**: create a new PR 🙃 

**Anything else we need to know?**:

/sig contributor-experience

cc @cblecker @nikhita 

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

/kind bug

### PR DESCRIPTION
**Pull request by paralika**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
It will fix PR templates.

**Which issue(s) this PR fixes**:

Fixes #71685 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by castrojo** [comment]
/lgtm

Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label
 -->
 
 **What type of PR is this?**
-> Uncomment only one, leave it on its own line:
+> Uncomment only one " /kind <>" line, hit enter to put that in a new line, and remove leading whitespaces from that line:
```
Comment (dixudx) [inline .github/PULL_REQUEST_TEMPLATE.md]: Use \` /kind <>\` here to emphasize this?

Comment (paralika) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good point. Done.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Fixes #
 
 **Does this PR introduce a user-facing change?**:
 <!--  
-If no, just write "NONE".
+If no, just write "NONE" in the release note block below.
```
Comment (krmayankk) [inline .github/PULL_REQUEST_TEMPLATE.md]: nit: should we just say `release-note` instead of the `release note`

**Pull request merged**